crates/analyzer/tests/errors.rs

```diff
@@ -355,5 +355,5 @@ test_file! { invalid_struct_pub_qualifier }
 test_file! { mut_mistakes }
 test_file! { invalid_comparisons }
 
-test_file! { test_call }
-test_file! { test_params }
+// test_file! { test_call }
+// test_file! { test_params }
```

Comment:
@g-r-a-n-t These tests aren't producing snapshots, which breaks the test on wasm. I assume there's no error output? (The `test_file` macro should probably assert that there's error output)

Reply:
yeah, looks like insta just does this

```
#[test]
fn test_my_thing() {
    assert_snapshot!("test123")
}
```

```
Snapshot file: crates/analyzer/tests/snapshots/errors__my_thing.snap
Snapshot: my_thing
Source: crates/analyzer/tests/errors.rs:101
```